plugins/mcreator-core/blockly/toolboxes/toolbox_procedure.xml

```diff
@@ -56,11 +56,20 @@
         <block type="text_join"/>
         <block type="text_print"/>
         <block type="text_length"/>
+        <block type="text_index_of">
+            <value name="from">
+                <block type="math_number">

```

Comment:
This could probably be a shadow block to avoid this extra value when not needed (or alternatively there could be another version of this block without start index).

Reply:
Another block is redundant approach imo